src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1580,6 +1581,18 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
             }
         }
 
+        if (!empty($config['default_parameters'])) {
```

Comment:
we try to never use empty in the codebase

Reply:
OK. Changed for a `[] === ` comparison.